Services/MainMenu/classes/Items/class.ilMMItemRepository.php

```diff
@@ -173,7 +174,7 @@ public function getItemFacade(IdentificationInterface $identification = null) :
         if ($identification === null || $identification instanceof NullIdentification || $identification instanceof NullPluginIdentification) {
             return new ilMMNullItemFacade($identification ? $identification : new NullIdentification(), $this->main_collector);
         }
-        if ($identification->getClassName() === ilMMCustomProvider::class) {
+        if ($identification->getClassName() === ilMMCustomProvider::class || $identification instanceof PluginIdentification) {
```

Comment:
@sragfw technically speaking, plugin items are not custom icons, so the check for `PluginIdentification` would be wrong here. Probably better would be to do another `ItemFacade` for all items of type `Lost`. In this facade one would implement that they are deleteable (only the information about the item in `ilMMItemStorage` and `ilMMItemTranslationStorage` can be deleted). would you have time to implement such a facade?

Reply:
@chfsx I think, I found an other possible fix to check `isDeletable` and not `isCustom`

Reply:
@chfsx Sorry, but I have currently no time to implement a new facade for lost items

For instance I don't get based on `$identification`, if the item is `Lost` ...